ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -334,7 +342,18 @@ TFLiteInterpreter::loadModel (int num_threads, accl_hw accelerator)
   }
 
   /** set delegate after the accelerator prop */
-  if (accelerator == ACCL_CPU_NEON || accelerator == ACCL_NPU) {
+  if (accelerator == ACCL_CPU_NEON) {
+#ifdef TFLITE_XNNPACK_DELEGATE_SUPPORTED
+    /* for xnnpack delegate */
+    TfLiteXNNPackDelegateOptions xnnpack_options = TfLiteXNNPackDelegateOptionsDefault();
+    xnnpack_options.num_threads = num_threads;
+
+    xnnpack_delegate.reset (TfLiteXNNPackDelegateCreate (&xnnpack_options));
+    setDelegate (xnnpack_delegate.get ());
+#else
+    ml_logw ("XNNPACK delegate support is available only in Android with tflite v2.3.0 or higher");
```

Comment:
The logic is changed weirdly. Double check. (e.g., NNAPI_DELEGATION with accerlator == ACCL_CPU_NEON)